cypress/plugins/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Is this file needed? Will it be used anywhere?

Reply:
Yes, it will be used. We can add more configuration on this file as we write more tests. I have kept it to populate the structure.